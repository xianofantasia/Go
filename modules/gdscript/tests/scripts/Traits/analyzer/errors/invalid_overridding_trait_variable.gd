trait SomeTrait:
	var some_variable: Node

class SomeClass:
	uses SomeTrait
	var some_variable: Node2D

func test():
	print("ok")
