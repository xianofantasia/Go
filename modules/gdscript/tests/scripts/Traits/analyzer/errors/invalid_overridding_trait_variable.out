GDTEST_ANALYZER_ERROR
>> ERROR at line 6: "some_variable" variable type should be "Node" as declared in trait "SomeTrait".
